src/functions.php

```diff
@@ -420,3 +420,37 @@ function env($name = null, $value = null)
         return null;
     }
 }
+
+/**
+ * Adds a global argument
+ *
+ * @param string $name
+ * @param array  $argument
+ *
+ * @return mixed
+ */
+function argument($name, array $argument = null)
+{
+    if (null === $argument) {
+        return Deployer::get()->getInput()->getArgument($name);
```

Comment:
Better use `input()` than `Deployer::get()->getInput()`. It takes `Input` from current tasks context.